tests/HUBViewModelDiffTests.m

```diff
@@ -184,10 +184,68 @@ - (void)testComplexChangeSet
     XCTAssert([diff.insertedBodyComponentIndexPaths containsObject:[NSIndexPath indexPathForItem:2 inSection:0]]);
     XCTAssert([diff.insertedBodyComponentIndexPaths containsObject:[NSIndexPath indexPathForItem:9 inSection:0]]);
     XCTAssert([diff.reloadedBodyComponentIndexPaths containsObject:[NSIndexPath indexPathForItem:1 inSection:0]]);
-    XCTAssert([diff.reloadedBodyComponentIndexPaths containsObject:[NSIndexPath indexPathForItem:7 inSection:0]]);
+    XCTAssert([diff.reloadedBodyComponentIndexPaths containsObject:[NSIndexPath indexPathForItem:8 inSection:0]]);
```

Comment:
The implementation we had made some incorrect assumptions about the index paths you need to pass to `UICollectionView performBatchUpdates:completion:` so would sometimes generate incorrect index paths that caused the crash.

The new code generates correct index paths, so this test has to be updated to reflect the correct logic.